### DISCUSSION TITLE
Deploy Serverpod to own Docker Server?

### DISCUSSION THREAD

**Discussion by robertapache** [serverpod/serverpod#691] [Q&A] [answered]
Hi, I am on day 5 of my Serverpod journey. Having built my first Serverpod Rest Services, I would like to deploy this to my Docker (mostly Test) Server. 
By now, I didn't find any Documentation on this. Do you think this is a good idea?  

Greetings from rainy northern Germany,
Robert

**Comment by vlidholt** [comment, marked as answer]
Hi! Congrats on starting your Serverpod journey! 🚀 You can deploy Serverpod to anywhere where you can run Dart and a Docker container is a great choice. Will will bring better support for this in version 1.1, but you can find a working Docker file (place it in your <myproject>_server directory) here already: https://github.com/serverpod/gcp/blob/main/gcp_server/Dockerfile

**Reply by robertapache** [reply]
Excellent Stuff. Thanks!
